_includes/analytics/cloudflare.html

```diff
@@ -0,0 +1,7 @@
+<!-- Cloudflare Web Analytics -->
+<script
+  defer
+  src="https://static.cloudflareinsights.com/beacon.min.js"
+  data-cf-beacon="{'token': '{{ site.analytics.cloudflare.id }}'}"
```

Comment:
Please note the use of single/double quotes

```suggestion
  data-cf-beacon='{"token": "{{ site.analytics.cloudflare.id }}"}'
```